doc/source/reference/arrays.scalars.rst

```diff
@@ -191,29 +191,31 @@ Inexact types
    This means that variables with equal binary values but whose datatypes are of
    different precisions may display differently::
```

Comment:
Although maybe it's just the use of `::` here that's problematic.

Reply:
Yes, I noticed this issue while working on #26745 in many places. Would it be better to handle such changes here? It would reduce the diff there, but I think there's no way to test the rendering of the examples out without having integrated `jupyterlite-sphinx`, so we might as well test it there better at the expense of a larger diff.

Reply:
If the plan is to merge this PR and then that one, I'd prefer to not knowingly make some examples not render very nicely. Just in case they get missed or that PR takes a long time to land for whatever reason. I don't mind making the diff a bit longer to sort out that, I think for this specific example all that's needed is to delete the `::` and it will automatically get the nice example markup you changed the docs to use elsewhere in this PR.